classes/trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+# ./classes/trimmer_decorator.rb
+require './classes/base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.strip
+  end
```

Comment:
- Good job done so far, but according to the project requirements, this method should make sure that the output of `@nameable.correct_name` has a maximum of 10 characters. If it's longer it should trim the word. Please fix this 🙏

Reply:
Thank you @mckent05